cloudevents/sdk-python
# More pythonic interface for CloudEvents

- Link to issue this resolves
#33 
#26 

- What I did
I made a class which internalizes the HTTP marshallers, verifies required fields, and currently supports HTTP post requests to notify event listeners on other endpoints.
 
- How I did it
I made an Event class with an internal v1.Event, and an internal HTTPMarshaller

- How to verify it
I will make tests in a different PR, but you can run the sample code as shown below:

`pip3 install ./sdk-python`
`cd sdk-python/samples/python-event-requests/`
`python3 sample-server.py`

In a different terminal but still in the same samples/python-event-requests folder:

`python3 cloudevent_to_request.py http://localhost:3000/event`

If the sample code worked you'll see the same event printed in both terminals

- One line description for the changelog
Outlined a more Pythonic Event handler
